### ISSUE TITLE
Loop Detection in a LinkedList 

### ISSUE THREAD

**Issue by Maruti54** [TecHAyusH6476/DSA-Prep-HacktoberFest22#50]
Please assign me this issue

**Comment by TecHAyusH6476** [comment]
@Maruti54 Yeah sure but please make sure to commit the files after verifying it
Happy Hacktober !!

**Comment by TecHAyusH6476** [comment]
@Maruti54 Assigned

**Comment by priyanshuanand166** [comment]
Assign this to me as i am well versed in dsa and i am pretty confident that you will like my merge request.

**Comment by ishita126jain** [comment]
Hey can you assign this issue to me so i can contribute in your repo @TecHAyusH6476

**Comment by Rishi0510** [comment]
Hello! Can you please assign me this under Hacktoberfest? @Maruti54

**Comment by Avy2808** [comment]
@Maruti54  can you please assign this issue to me under hacktober fest 2022 ?

**Comment by falguni300103** [comment]
Can you please assign this issue to me? @TecHAyusH6476

**Comment by TecHAyusH6476** [comment]
@ishita126jain @Rishi0510 @Avy2808 @falguni300103 @ayushprajapati0706 
I have assigned you guys Please make sure you are creating separate directories and dont put the same algos if its already present. Thank You!

**Comment by ishita126jain** [comment]
Hey @TecHAyusH6476 I have done my task can you please checkout my PR I have
added a cpp code to detect the loop on the linked list
Thanks for this opportunity.

**Comment by TecHAyusH6476** [comment]
Yeah I have merged the PR 
Thanks for contributing!

**Comment by ishita126jain** [comment]
Thank you for your response

**Comment by Dishu099** [comment]
Hey! @Maruti54 I want to contribute in your repository so please assign me this task.

**Comment by hitali-khachane** [comment]
Please Assign me this .I would make sure to provide best code with better time complecity
